src/saturn_engine/worker/services/loggers/logger.py

```diff
@@ -29,6 +29,7 @@ def executable_message_data(
     return pipeline_message_data(xmsg.message, verbose=verbose) | {
         "job": xmsg.queue.name,
         "input": xmsg.queue.definition.input.name,
+        "owner": xmsg.message.message.metadata.get("owner", ""),
```

Comment:
Following my comment above, I would replace it with `if (labels := xmsg.message.message.metadata.get("labels")): data["labels"] = labels.copy()`